### DISCUSSION TITLE
Suporte a subsubsection

### DISCUSSION THREAD

**Discussion by wcmjunior** [discentesppgcc/ppgccufmg#2] [Q&A] [answered]
Existe algum motivo particular pelo qual a classe atual não suporta numeração de `subsubsection`? Meu questionamento se deve ao fato de que a classe anterior do ppgcc suportava e eu preciso portar um documento da classe antiga para a nova.

Classe nova: https://github.com/discentesppgcc/ppgccufmg/blob/main/ppgccufmg.cls#L501

Classe antiga:
```
...
\maxsecnumdepth{subsubsection}
...
```

Se eu efetuar a numeração de `subsubsection`, haverá algum problema de não conformidade?

**Comment by ramongonze** [comment, marked as answer]
Olá Wilson,

Não existe nenhum motivo particular para tal.
Abri o issue #3 para adicionar o suporte a ```subsubsection```.

**Reply by wcmjunior** [reply]
Muito obrigado pela velocidade em responder e implementar a solução. :-)
